client/oci_env/commands.py

```diff
@@ -158,4 +166,9 @@ def profile(args, client):
             exit_with_error(f"{args.profile} doesn't have a README.md")
 
 def poll(args, client):
-    client.poll(args.attempts, args.wait)
\ No newline at end of file
+    client.poll(args.attempts, args.wait)
+
+
+def edit_env(args):
+    editor = os.getenv("OCI_ENV_EDITOR", "vim")
```

Comment:
I think "EDITOR" is a wildly used variable for this. Also xdg-open seems like a very good option. WDYT?

https://stackoverflow.com/questions/19252791/how-to-open-the-default-text-editor-in-linux